config/vars.template.json

```diff
@@ -7,5 +7,7 @@
   "OFFICE_STATUS_CHANNEL_ID": "CHANNEL_ID",
   "RESUME_CHANNEL_ID": "CHANNEL_ID",
   "IRC_USER_ID": "USER_ID",
-  "MOD_USER_ID_FOR_BAN_APPEAL": "USER_ID"
+  "MOD_USER_ID_FOR_BAN_APPEAL": "USER_ID",
+  "CODEY_COIN_ROLE_ID": "ROLE_ID",
+  "CODEY_COIN_ROLE_NAME": "ROLE_NAME"
```

Comment:
This should be unnecessary, since a role id uniquely identifies a role.

Reply:
Removed it from here.